backend/licenses-core/src/main/java/org/eclipse/sw360/licenses/db/LicenseDatabaseHandler.java

```diff
@@ -1164,21 +1164,29 @@ public RequestStatus deleteObligations(String id, User user) throws SW360Excepti
         }
     }
 
-    public RequestStatus deleteLicenseType(String id, User user) throws SW360Exception {
+    public RequestStatus deleteLicenseType(String id, User user) {
         LicenseType licenseType = licenseTypeRepository.get(id);
-        assertNotNull(licenseType);
 
-        // Remove the license type if the user is allowed to do it by himself
-        if (PermissionUtils.isUserAtLeast(UserGroup.SW360_ADMIN, user)) {
-            if (checkLicenseTypeInUse(id) > 0) {
-                return RequestStatus.IN_USE;
-            }
-            licenseTypeRepository.remove(licenseType);
-            return RequestStatus.SUCCESS;
-        } else {
-            log.error(user + " does not have the permission to delete license type.");
+        if (licenseType == null) {
+            log.warn("License type not found with ID: {}", id);
+            return RequestStatus.FAILURE;
```

Comment:
`RequestStatus.INVALID_INPUT` would be more appropriate response as `FAILURE` can be a 500 error as well.